### ISSUE TITLE
Auto-load resource JARs for LanguageTool

### ISSUE THREAD

**Issue by reckart** [dkpro/dkpro-core#750]
Auto-load resource JARs for LanguageTool through a model provider. This is currently blocked by the way that LT loads resources (https://github.com/languagetool-org/languagetool/pull/351).

**Comment by reckart** [comment]
Probably won't do because nobody as time to work on languagetool-org/languagetool#351. Removing version tag for now.
